src/lib/cli.js

```diff
@@ -18,16 +19,19 @@ program
   .option('-r, --require <path>', 'Require location for support files (repeatable)', collect, [])
   .option('-c, --cucumber <path>', 'Path to specific cucumber installation')
   .option('-w, --workers <int>', 'Number of workers (Defaults to # Processors)', parseInt)
-  .option('-l, --logdir <path>', 'Output dir for test logs')
-  .option('-s, --silentsummary', 'Silences summary output so it can be handled via the returned promise')
+  .option('-l, --log-dir <path>', 'Output dir for test logs. Default: ' + defaultOptions.logDir)
+  .option('-s, --silent-summary', 'Silences summary output so it can be handled via the returned promise')
   .option('-v, --verbose', 'Adds verbose output to console')
-  .option('-i, --inlinestream',
+  .option('-i, --inline-stream',
     'Inlines stream in real time in addition to multi-cuke output. ' +
     '*Note* This adds complexity to the logs that are hard to decipher, but included if needed for debugging'
   )
-  .option('-d, --devMode', 'Shortcut for running cucumber-js directly')
+  .option('-d, --dev-mode', 'Shortcut for running cucumber-js directly')
```

Comment:
I'm definitely aligned with consistency for these flags and do think kebab case is the right move here, but removing this backwards compatibility feels unnecessary at the moment. Since this arg parsing isn't tied to a command, I think we could do some smart stuff such as keep these changes above as additions, and also keep the old  in a commented block like:
```
...
.option('--strict', 'Fail fast if a step is undefined')
// Deprecated flags to be removed in v1.0.0
.option('-l, --logdir <path>', 'Output dir for test logs')
.option('-d, --devMode', 'Shortcut for running cucumber-js directly')
(etc) ...
```
And then in the export:
```
export const args = {	
  requires: (program.require.length) ? program.require : undefined,
  cucumberPath: program.cucumber,		 
  workers: program.workers,	
  logdir: program.logdir || program.logDir,
  silentSummary: program.silentsummary || program.silentSummary,
  ...
```

This basically also buys us some deprecation notices, in case anyone is using this in their CI:
(Before the export, after `.parse(process.argv)`:
```
let deprecations = [
  { oldFlag: '--logdir', newFlag: '--log-dir'},
  { oldFlag: '--devMode', newFlag: '--dev-mode'}
  ...
];
deprecations.forEach(flag => {
  if(program.rawArgs.includes(flag.oldFlag)) {
    console.warn(`${flag.oldFlag} is deprecated and will be removed in future versions. Please use ${flag.newflag} instead`);
  }
});
```